src/lib/AutomateSDK.ts

```diff
@@ -203,12 +203,13 @@ export class AutomateSDK {
 
   public async createTask(
     _args: CreateTaskOptions,
-    overrides: Overrides = {}
+    overrides: Overrides = {},
+    authToken?: string
   ): Promise<TaskTransaction> {
     const args = await this._processModules(_args);
 
     // Ask for signature
-    if (!this._token) await this._requestAndStoreSignature();
+    if (!authToken) await this._signature.getAuthToken();
```

Comment:
Should it be 
```ts
    if (!authToken) authToken = await this._signature.getAuthToken();
```

Reply:
It's not strictly necessary as getting it will be enough to put it in local cache for next requests.
But indeed that will look better and more explicit so I'll do your suggested change